social_core/backends/open_id_connect.py

```diff
@@ -45,38 +53,49 @@ class OpenIdConnectAuth(BaseOAuth2):
     JWKS_URI = ''
     JWT_ALGORITHMS = ['RS256']
     JWT_DECODE_OPTIONS = dict()
+    TOKEN_ENDPOINT_AUTH_METHOD = ''
```

Comment:
Shouldn't this be by default `'client_secret_post'`?

Reply:
Agreed. The logic says "anything other than client_secret_basic is client_secret_post" (hence the boolean function `use_basic_auth` in oauth.py), but it doesn't harm to document it properly.

Reply:
Ok, I guess I agree. Can you maybe just copy paste your own comment `unspecified, let the server choose via openid autoconfiguration` and add that as a comment? I guess in that case it's clear why it's empty.

Reply:
I guess so, although the same also applies to all these parameters:

```
    ID_TOKEN_ISSUER = ''
    ACCESS_TOKEN_URL = ''
    AUTHORIZATION_URL = ''
    REVOKE_TOKEN_URL = ''
    USERINFO_URL = ''
    JWKS_URI = ''
```

Reply:
I have added a comment covering all these options

Reply:
Awesome, thanks!